### ISSUE TITLE
Search for a part of street-name doesn't return results

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#5277]
**[Submitted to the original trac issue database at 9.32am, Sunday, 15th February 2015]**

In the middle of Kyiv, Ukraine, there is a street called "  " (str. of Architect Gorodetskiy).

I searched for "", and didn't have any search result.

Thanks!

**Comment by tomhughes** [comment]
**Author: lonvia**
**[Added to the original trac issue at 1.07pm, Sunday, 15th February 2015]**

Nominatim has problems distinguishing between  and . Searching for "" will turn up the street if you search for more results.

Duplicate of #2758.

**Comment by tomhughes** [comment]
**Author: k-bx@k-bx.com**
**[Added to the original trac issue at 1.14pm, Sunday, 15th February 2015]**

@lonvia too bad to hear that nominatim has these kind of problems. Another thing is that it can't find anything by term  also. Should I reopen ticket?

**Comment by tomhughes** [comment]
**Author: lonvia**
**[Added to the original trac issue at 1.37pm, Sunday, 15th February 2015]**

It does find it, but only after 'More results' because there is an exact match. That is normal.
